uadapy/plotting/plots1D.py

```diff
@@ -129,6 +145,7 @@ def plot_1d_distribution(
         colorblind_safe=False,
         show_plot=False,
         dot_size=0,
+        restrict_to_1d=False,
```

Comment:
I think, it would be better, if the default hier is also True (consistent with everything else). Could you provide a function plot_multidim_distribution_1d that does the same thing as the current function?

Reply:
Addressed in commit https://github.com/UniStuttgart-VISUS/uadapy/pull/32/commits/4e5163adc21d9f407de1ddef580982c02f837b1a